src/components/plugins/VgtColumnDropdown.vue

```diff
@@ -0,0 +1,53 @@
+<template>
```

Comment:
@xaksis I think this is the best we can do to separate features from the core functionality.
My idea was to create a plugins folder, and add some "example" components there, and expose these to the user.
We could create cookbook section in de docs, that explains this.

Imo I think we should gradually move some of the components to the plugins folder. So the user can use those, extend from it. Or drop in something entirely different.

If you want to move to something that's for ex renderless, that's possible. But that would require a total rewrite. And imo I think Vue3's composition API could shine there. Abstracting out render related functions.

I think you can decrease maintenance costs, by utilizing scoped slots more, and having some example components in plugins. That way, you could put the MVP in plugins, while offering a lot of flexibility to the user.